server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailSetMethodContract.scala

```diff
@@ -3473,15 +3473,15 @@ trait EmailSetMethodContract {
          |          "htmlBody": [
          |            {
          |              "partId": "a49d",
-         |              "type": "text/html"
+         |              "type": "text/html",
+         |              "header:Specific:asText": "MATCHME"
```

Comment:
This shall be a property of all `EmailBodyPart` objects so also of `attachments`, `textBody` and `bodyStructructure`

Can you add tests for those?